doc/plugins/proxy-rewrite.md

```diff
@@ -26,7 +27,12 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
             "uri": "/test/home.html",
             "scheme": "http",
             "host": "iresty.com",
-            "enable_websocket": true
+            "enable_websocket": true,
+            "headers": {
+                "X-Api-Version": "v1",
+                "X-Api-Engine": "apisix",
+                "X-Api-Test": "null"
```

Comment:
the `"null"` is not good, if the string value is `"null"`, how to deal with it?
it should be `null`.

Reply:
empty string is good, l revise it.

Reply:
$ resty -e 'print(require("cjson").encode({a=ngx.null}))'
{"a":null}

how about we use `null` ?

Reply:
@moonming What about you?

Reply:
I think empty strings are better.

Reply:
I think we should keep the same style with OpenResty API:
```
 ngx.req.set_header("X-Foo", nil)
```
And I am not sure we should use `null` or `cjson.null` here, you can make a test.